### PR TITLE
Feature/boas 971 big split clean up

### PR DESCRIPTION
**Pull request by francescobrenci-AR**
## Describe your changes

Remove all APPEALS references from NSIP APPLICATIONS App

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/testing/appeals/factory/address.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { faker } from '@faker-js/faker';
 import { fake } from '@pins/platform';
 
-/** @typedef {import('@pins/api').Schema.Address} AddressData */
+/** @typedef {import('@pins/applications.api').Schema.Address} AddressData */
```
Comment (francescobrenci-AR) [inline appeals/web/testing/appeals/factory/address.js:4]: @fabfab-kc some definitions are still imported from Applications - maybe worth copying them over to Appeals

**Review by fabfab-kc — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -27,14 +24,18 @@
...
     "test": "turbo run test",
-    "tscheck": "turbo run prisma-generate --concurrency=1 && turbo run tscheck --parallel --filter=!@pins/api --filter=!@pins/appeals.api",
-    "web": "npm run dev --workspace=@pins/web",
+    "tscheck": "turbo run prisma-generate --concurrency=1 && turbo run tscheck --parallel --filter=!@pins/applications.api --filter=!@pins/appeals.api",
     "dev:appeals": "turbo run dev --filter=./appeals/* --no-cache --parallel --continue",
     "test:appeals": "turbo run test --filter=./appeals/*",
     "web:appeals": "npm run dev --workspace=@pins/appeals.web",
-    "api:appeals": "npm run dev --workspace=@pins/appeals.api"
+    "api:appeals": "npm run dev --workspace=@pins/appeals.api",
+		"dev:applications": "turbo run dev --filter=./apps/* --no-cache --parallel --continue",
```
Comment (beejjacobs) [inline package.json:34]: Is the spacing off here? prettier should've formatted it, odd.

**Review by beejjacobs — DISMISSED** [review]
This looks good to me - lots of deleting! Most of it boils down to find and replace of @pins/api -> @pins/applications.api - good to be consistent between the two, so I like that.

Really my only comment, is personally I think the npm scripts would be better prefixed with the domain, e.g.

- `appeals:api` 
- `appeals:web`
- `applications:api`
- etc....

But I want to get other views on that, not a big deal either way.

File: package.json (hunk 2 above)
Comment (francescobrenci-AR) [reply package.json:34]: yeah - I can't see the spacing in the local file

Comment (beejjacobs) [reply package.json:34]: I think it's tabs verses spaces, and if you have tabs set to 2 space width, it'll look the same.

Our prettierrc has useTabs, but the package json seems to have spaces in. Odd!

**Review by beejjacobs — APPROVED** [review]
Looks good to me. Perhaps we get a tester on dev once merged to check we've not broken anything?

**Review by beejjacobs — APPROVED** [review]

**Pull request merged**